articles/multifactor-authentication/custom.md

```diff
@@ -0,0 +1,130 @@
+
+::: note
+The MFA settings defined in rules will always take precedence over the toggles in the MFA section of the Dashboard.
+:::
+
+## The provider setting
+
+The `provider` setting now defaults to `any`. This allows the toggles on the [Dashboard > MFA](${manage_url}/#/mfa) to control which providers are enabled, and the user to select from any of the enabled providers. 
+
+You can also set the `provider` to any of these legacy options, which are still valid to ensure backwards compatibility:
+
+* `guardian` = uses SMS or Push via Guardian
+* `google-authenticator` = forces you to use a UI specific to Google Authenticator
+* `duo` = forces you to use Duo
+
+Setting the `provider` manually will override the enabled/disabled toggles in the Dashboard, and is not the recommended way to control which factors are used.
+
+```js
+function (user, context, callback) {
+
+  // Forcing the provider to Guardian programmatically
+  context.multifactor = {
+    provider: 'guardian'
+  };
+
+  callback(null, user, context);
+}
+```
+
+## Implementing contextual MFA
+
+The exact requirements for configuring Contextual MFA will vary. Below are sample snippets you might consider using as you customize your specific solution.
+
+### Customize MFA for select users
+
+You may customize MFA to run only for users who are authenticating against specific applications in your tenant, or only for users who are marked to use MFA.
+
+```js
+function (user, context, callback) {
+
+  //var CLIENTS_WITH_MFA = ['REPLACE_WITH_YOUR_CLIENT_ID'];
+  // run only for the specified applications
+  // if (CLIENTS_WITH_MFA.indexOf(context.clientID) !== -1) {
+    // uncomment the following if clause in case you want to request a second factor only from user's that have user_metadata.use_mfa === true
+    // if (user.user_metadata && user.user_metadata.use_mfa){
+      context.multifactor = {
+        provider: 'any',
+        allowRememberBrowser: false
+      };
+    // }
+  //}
+
+  callback(null, user, context);
+}
+```
+
+If you choose to selectively apply MFA, you will need the appropriate `clientID` values, and the code will be executed as part of a [Rule](/rules) whenever a user logs in.
+
+More specifically, you will uncomment and populate the following line of the rule template above with the appropriate client IDs:
+
+`var CLIENTS_WITH_MFA = ['REPLACE_WITH_CLIENT_ID'];`
+
+By setting `allowRememberBrowser: false`, the user will always be prompted for MFA when they login. This prevents the browser cookie from saving the credentials and helps make logins more secure, especially from untrusted machines. See [here](#change-the-frequency-of-authentication-requests) for details.
+
+### Change the frequency of authentication requests
+
+When using the Guardian multi-factor application, by default users are given the option to be remembered and skip MFA for a period of 30 days. To disable this choice for users, set the `allowRememberBrowser` field to `false`.
```

Comment:
Minor: 'period of 30 days' => 'a period up to 30 days'. Maybe it doesn't worth the explanation, but currently you are remember for 7 days and up to 30 days. If you don't login in 7 days, your cookie will be come invalid, if you do, it will be renewed for 7 more days, up to 30.